work-with-us.html

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Work With Us
+permalink: /work-with-us/
+intro: We're not actively hiring right now, but we're always interested in meeting future candidates. Feel free to reach out! :)
+---
+
+<section class="work-with-us">
+	<div class="container-fluid hero-title">
+		<div class="col-md-12">
+			<header class="post-header">
+				<h1 class="post-title">Work With Us</h1>
+			</header>
+		</div>
+	</div>
+    <div class="post">
+        <div class="container-fluid">
+            <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+                    <h2>Who Are We?</h2>
+                        <p>We're a small team working to make foreign language teaching and learning more awesome. We’re passionate about education, equity, and technology. Our mission is to build a tool that liberates foreign language teachers from drilling vocabulary and grammar so that they can engage their students in meaningful communicative activities. We believe that taking a foreign language in middle or high school should give students the real world skills they need to communicate in a new language and we’re building the tools to get them there!</p>  
+                    <h2>Our Values</h2>
+                    <ul>
```

Comment:
Looking at this again, I just realized that these nested lists could be made simpler and may be one of the few cases for a description list (`dl` tag). It's used for associating a term to a particular definition.

[Read here on the html5doctor](http://html5doctor.com/the-dl-element/)

[Try it here](http://www.w3schools.com/html/tryit.asp?filename=tryhtml_lists_description)